app/Rest/Resources/UserResource.php

```
@@ -58,8 +58,9 @@ public function relations(RestRequest $request): array
     {
         return [
             HasMany::make('posts', PostResource::class),
+            BelongsToMany::make('saved_posts', PostResource::class),
```

PHPMD (2.15.0)

Avoid using static access to class '\Lomkit\Rest\Relations\BelongsToMany' in method 'relations'. (Clean Code Rules)

(StaticAccess)